candi/openapi/node_group.yaml

```diff
@@ -79,11 +79,12 @@ spec:
                 name:
                   type: string
                   pattern: "^[a-z0-9]([-a-z0-9]*[a-z0-9])?$" # In Amazon cloud, AWSMachineClass name can't contain dots.
-                  maxLength: 42 # One of labels assigned by the cloud instance manager contains full node name,
-                                # the same time, kubernetes labels keys and values can't be longer than 63 characters.
-                                # Length of hashes is 21 characters, so we leave 63-21=42 characters.
-                                # Further, validation webhook will check that length of <cluster prefix>-<group node name>
-                                # fits the 42 characters limitation.
+                  maxLength:
+                    42 # One of labels assigned by the cloud instance manager contains full node name,
```

Comment:
I don't think it's a good idea - to move the value on the next line

Reply:
All statuses written earlier were in English. Should I rewrite statuses to the Russian language for the rest?